app/src/pages/inside/projectSettingsPageContainer/content/elements/notificationRuleContent/notificationRuleContent.jsx

```diff
@@ -88,23 +98,32 @@ export const NotificationRuleContent = ({ item }) => {
     [LAUNCH_CASES.TO_INVESTIGATE]: formatMessage(messages[LAUNCH_CASES.TO_INVESTIGATE]),
   };
 
+  const attributesFieldText =
+    attributes.length > 1
+      ? `${formatMessage(messages.attributesLabel)} ${
+          attributesOperator === ATTRIBUTES_OPERATORS.AND
+            ? formatMessage(messages.attributesLabelAll)
+            : formatMessage(messages.attributesLabelAny)
```

Comment:
`formatMessage(attributesOperator === ATTRIBUTES_OPERATORS.AND ? messages.attributesLabelAll : messages.attributesLabelAny)`